vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -93,7 +93,7 @@ $global:map = @{
         $defaultContainerdPackageUrl
     );
     "c:\akse-cache\csi-proxy\"    = @(
-        "https://acs-mirror.azureedge.net/csi-proxy/v0.2.2/binaries/csi-proxy-v0.2.2.tar.gz"
+        "https://acs-mirror.azureedge.net/csi-proxy/v1.0.0/binaries/csi-proxy-v1.0.0.tar.gz"
```

Comment:
@AbelHu we do not store two versions in windows image?

Reply:
Agree. cc @mainred we need to cache this in new VHD with 8B

Reply:
it's done